code/__DEFINES/is_helpers.dm

```diff
@@ -38,7 +38,7 @@
 #define isplatingturf(A) (istype(A, /turf/open/floor/plating))
 
 //Mobs
-#define isliving(A) (istype(A, /mob/living))
+#define isliving(A) (is_type_in_typecache(A, GLOB.typecache_living))
```

Comment:
I'm dubious of this.

Reply:
I am also unclear of how this works

Reply:
I'm not unclear, I'm the one who 'discovered' typecaches, I'm just doubtful that they win out against a single ```istype()```, the goal of typecaches and ```is_type_in_typecache()``` was to stop people doing ```istype()```s inside of loops (such as the old ```is_type_in_list()```) since that IS dreadfully slow.

While assoc (and therefore typecache) lookup is O(log n) I'd hope ```istype()``` would be atleast that cheap.